src/main/java/teammates/ui/controller/StudentProfilePictureEditAction.java

```diff
@@ -113,23 +107,42 @@ private void uploadFileToGcs(byte[] transformedImage) throws IOException {
         return null;
     }
 
-    private Image getTransformedImage(Double leftX, Double topY, Double rightX, Double bottomY) {
+    private Image getTransformedImage() {
         Assumption.assertNotNull(_blobKey);
 
         Image oldImage = ImagesServiceFactory.makeImageFromBlob(_blobKey);
-        CompositeTransform finalTransform = getCompositeTransformToApply(leftX, topY, rightX, bottomY);
+        CompositeTransform finalTransform = getCompositeTransformToApply();
         OutputSettings settings = new OutputSettings(ImagesService.OutputEncoding.PNG);
 
         return ImagesServiceFactory.getImagesService().applyTransform(finalTransform, oldImage, settings);
     }
 
-    private CompositeTransform getCompositeTransformToApply(Double leftX, Double topY, Double rightX,
-                                                            Double bottomY) {
-        Transform crop = ImagesServiceFactory.makeCrop(leftX, topY, rightX, bottomY);
-        Transform resize = ImagesServiceFactory.makeResize(150, 150);
+    private Transform getScaleTransform() {
+        Double width = Double.parseDouble(_widthString);
+        Double height = Double.parseDouble(_heightString);
+        return ImagesServiceFactory.makeResize((int)Math.round(width), (int)Math.round(height));
```

Comment:
ok. I dont know how it works, but my only concern is if the image they upload is high resolution and they are unable to include all their facial features using a box that is fixed to be 150px by 150px. I believe it would be more robust to implement the policy at the backend and not on the front-end